### PR TITLE
Use utf-8-sig encoding for vk_validation_error_messages

### PR DESCRIPTION
**Pull request by ziga-lunarg**
This was causing encoding issues on some systems

Related to #6334

### CODE REVIEW

File: scripts/vk_validation_stats.py [hunk 1]
```diff
@@ -633,7 +633,7 @@ def isKhr(f): return f.startswith('VK_KHR_')
     def export_header(self, filename):
         if verbose_mode:
             print("\n Exporting header file to: %s" % filename)
-        with open(filename, 'w', newline='\n', encoding='utf-8') as hfile:
```
Comment (spencer-lunarg) [inline scripts/vk_validation_stats.py:636]: can you add a comment for me here

`# The validusage.json has text that might cause encoding issues (see PR 6334)

**Review by spencer-lunarg — APPROVED** [review]
with comment

File: scripts/vk_validation_stats.py (hunk 1 above)
Comment (ziga-lunarg) [reply scripts/vk_validation_stats.py:636]: Done

**Comment by spencer-lunarg** [comment]
@ziga-lunarg can this be closed now?

**Pull request closed without merging**